core/lib/l1_contract_interface/src/i_executor/structures/commit_batch_info.rs

```diff
@@ -101,6 +103,72 @@ impl<'a> Tokenizable for CommitBatchInfo<'a> {
                 ),
                 // `systemLogs`
                 Token::Bytes(serialize_commitments(&self.0.header.system_logs)),
+            ]
+        }
+    }
+
+    pub fn into_tokens_calldata(&self) -> Token {
```

Comment:
same for `into_tokens_calldata` and `into_tokens_blobs`. Also note, that `CommitBatchInfo::into_token` must be backward compatible, i.e. for old batches it mustn't append pubdata with blob commitment